src/components/Main/Footer/Contact.tsx

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
-
-// TODO: Add top graphic (should be same as header, but flipped)
-// TODO: Add logo
-// TODO: Add contact
-// TODO: Add links (Brand Manual should redirect to Figma since we haven't coded it)
-// TODO: Make the gwcuva@gmail.com a link to automatically send an email
+import discordSocial from '../../../assets/images/discord-icon.svg'
+import instagramSocial from '../../../assets/images/instagram-icon.svg'
+import facebookSocial from '../../../assets/images/facebook-icon.svg'
+import twitterSocial from '../../../assets/images/twitter-icon.svg'
+import {Row, Col} from 'react-bootstrap';
 
 function Contact() {
   return (
     <div>
-      <h3>Contact</h3>
-      <p className="interactive">gwcuva@gmail.com</p>
-      <img src="#" alt="icon" />
+      <h3 style={{paddingBottom: 10}}><strong>Contact</strong></h3>
```

Comment:
instead of style, try these https://getbootstrap.com/docs/4.1/utilities/spacing/